### PR TITLE
feat/x86_64

### PR DESCRIPTION
**Pull request by gStart9**

### CODE REVIEW

**Comment by gStart9** [comment]
Also added a version bump in there

File: Makefile [hunk 1]
```diff
@@ -1,34 +1,45 @@
...
 
-image.tar: Dockerfile docker_entrypoint.sh health-check.sh configurator/target/aarch64-unknown-linux-musl/release/configurator
-	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --tag start9/spark-wallet/main:${EMVER} --build-arg SPARK_VERSION=$(SPARK_VERSION) --platform=linux/arm64 -o type=docker,dest=image.tar .
+$(PKG_ID).s9pk: manifest.yaml instructions.md scripts/embassy.js LICENSE docker-images/aarch64.tar docker-images/x86_64.tar
+	if ! [ -z "$(ARCH)" ]; then cp docker-images/$(ARCH).tar image.tar; fi
+	embassy-sdk pack
+
+docker-images/aarch64.tar: Dockerfile docker_entrypoint.sh
+	mkdir -p docker-images
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --tag start9/$(PKG_ID)/main:$(PKG_VERSION) --platform=linux/arm64 -o type=docker,dest=docker-images/aarch64.tar .
```
Comment (elvece) [inline Makefile:38]:
```suggestion
	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --tag start9/$(PKG_ID)/main:$(PKG_VERSION) --platform=linux/arm64 --build-arg PLATFORM=arm64 -o type=docker,dest=docker-images/aarch64.tar .
```

File: Makefile [hunk 2]
```diff
@@ -1,34 +1,45 @@
...
 
-configurator/target/aarch64-unknown-linux-musl/release/configurator: $(CONFIGURATOR_SRC)
-	docker run --rm -it -v ~/.cargo/registry:/root/.cargo/registry -v "$(shell pwd)"/configurator:/home/rust/src start9/rust-musl-cross:aarch64-musl cargo +beta build --release
+docker-images/x86_64.tar: Dockerfile docker_entrypoint.sh
+	mkdir -p docker-images
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --tag start9/$(PKG_ID)/main:$(PKG_VERSION) --platform=linux/amd64 -o type=docker,dest=docker-images/x86_64.tar .
```
Comment (elvece) [inline Makefile:42]:
```suggestion
	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --tag start9/$(PKG_ID)/main:$(PKG_VERSION) --platform=linux/amd64 --build-arg ARCH=x86_64 --build-arg PLATFORM=amd64 -o type=docker,dest=docker-images/x86_64.tar .
```

File: Dockerfile [hunk 3]
```diff
@@ -1,13 +1,20 @@
 FROM node:current-alpine3.12
 
-ARG SPARK_VERSION
+ARG PKG_VERSION
+# arm64 or amd64
+ARG PLATFORM
+# aarch64 or x86_64
+ARG ARCH
```
Comment (elvece) [inline Dockerfile:7]:
```suggestion
```

does not seem like this is actually needed in your Dockerfile, just the `PLATFORM`

**Review by elvece — APPROVED** [review]

**Comment by elvece** [comment]
Needs rebase and rename to reflect actual change, which is converting main procedure to JS.

**Review by elvece — CHANGES_REQUESTED** [review]
Needs rebase and rename to reflect actual change, which is converting main procedure to JS.

**Pull request closed without merging**